met-web/src/components/survey/edit/EditForm.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { useContext } from 'react';
+import { Grid, Stack, TextField } from '@mui/material';
+import { ActionContext } from './ActionContext';
+import { MetLabel, PrimaryButton, SecondaryButton } from 'components/common';
+import { SurveyFormProps } from '../types';
+
+export const EditForm = ({ handleClose }: SurveyFormProps) => {
+    const { handleSubmit, isSubmitting, submission } = useContext(ActionContext);
+
+    return (
+        <Grid
+            container
+            direction="row"
+            justifyContent="flex-start"
+            alignItems="flex-start"
+            spacing={2}
+            padding={'2em 2em 1em 2em'}
+        >
+            {submission?.comments.map((comment, index) => {
+                return (
+                    <Grid item xs={12} key={index}>
+                        <MetLabel>{comment.label}</MetLabel>
+                        <TextField
+                            defaultValue={comment.text}
+                            sx={{ width: '100%' }}
+                            multiline={true}
+                            rows={3}
+                            onChange={(e) => {
+                                comment.text = e.target.value;
+                            }}
+                        />
+                    </Grid>
+                );
+            })}
+            <Grid item container xs={12} direction="row" justifyContent="flex-end" spacing={1} sx={{ mt: '1em' }}>
```

Comment:
Have we considered mobile view here. If we needed to switch Secondary and Primary button positions for mobile, I've discovered this neat order property. Hope it works
![image](https://user-images.githubusercontent.com/91914654/209242582-96796061-8364-4314-8981-e560f92cf525.png)